File: fixtures/r_output/windows_get_with_escaped_values.R
require(httr)

data = list(
  `a` = 'b',
  `c` = 'd',
  `e` = 'f',
  `h` = 'i',
  `j` = 'k',
  `l` = 'm'
)

res <- httr::POST(url = 'http://www.url.com/page', body = data)

File: fixtures/rust_output/windows_get_with_escaped_values.rs
extern crate reqwest;

fn main() -> Result<(), reqwest::Error> {

    let res = reqwest::Client::new()
        .post("http://www.url.com/page")
        .body("a=b&c=d&e=f&h=i&j=k&l=m")
        .send()?
        .text()?;
    println!("{}", res);

    Ok(())
}